### ISSUE TITLE
Tutanota discards email being composed when switching apps

### ISSUE THREAD

**Issue by alstr** [tutao/tutanota#4419]
- [✅] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [✅] I've searched and did not find a similar issue.

**Bug in mobile app**

**Describe the bug**

Changing app whilst composing an email causes the current email contents to be discarded.
 
**To Reproduce**
1. Compose email.
2. Switch app e.g. camera to take photo attachment.
3. Switch back to Tutanota app.
4. App logs in again and email draft has been discarded.

(PS is it GitHub or iOS that is trying to add new numbered lines items every time I press return. Annoying as heck.)

**Expected behavior**

Tutanota should at the very least save the email to Drafts.

**Smartphone (please complete the following information):**
 - Device: iPhone XR
 - OS: iOS 15.5
 - Version 3.98.12

**Comment by charlag** [comment]
Hi
Is the problem that the app is kicked out of memory? Do you see the login screen/process when you switch back to the app?

**Comment by alstr** [comment]
Yes, I get the app log in screen when switching back to the app.

Not sure if it's being kicked out of memory - I don't close it and it stays open in the background.

**Comment by charlag** [comment]
If you see the login screen then it was effectively closed, yes.

This is not a bug per se but it's a missing local/autosave draft feature. I can't find the issue for it so I will convert this one.

**Comment by paw-hub** [comment]
Moving to Blocked until #9681 is fixed.
